specs/VsmAnalyticsVisibilityForAllUsers.spec

```diff
@@ -0,0 +1,91 @@
+VSMAnalytics
```

Comment:
Use the correct Specification and Scenario name in the file too, since this is what is used in the gauge reports. Use the name `VsmAnalyticsVisibilityForAllUsers.spec`